### PR TITLE
Fixing post card components and their usage

### PR DESCRIPTION
**Pull request by harrysolovay**

### CODE REVIEW

File: gatsby-node/constants.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const {map} = require('ramda');
 const {join, resolve} = require('path');
 
-const lifeCycleStages = ['createPages', 'onCreateNode'];
+const lifeCycleStages = ['onCreateNode', 'createPages'];
```
Comment (sammartinez) [inline gatsby-node/constants.js]: Why did these get flipped?

File: src/components/Card/Post/Condensed.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import {KASHMIR_BLUE_COLOR} from '~/constants';
+import {classNames} from '~/utilities';
+import asCard from '../asCard';
+import {Author} from './fragments';
+
+const styles = css`
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: ${KASHMIR_BLUE_COLOR};
+  * {
```
Comment (sammartinez) [inline src/components/Card/Post/Condensed.js]: This seems interesting? What if the background-color gets set to white as well?

File: src/components/Card/Post/Condensed.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+          </div>
+
+          <div className='author'>
+            <Author {...{to, name, handle, avatar}} />
+          </div>
+        </div>
+      </ConditionalAnchor>
+    );
+  },
+);
```
Comment (sammartinez) [inline src/components/Card/Post/Condensed.js]: Do we want unit tests for the new files?

File: gatsby-node/constants.js (hunk 1 above)
Comment (harrysolovay) [reply gatsby-node/constants.js]: Wanted them to be in execution order––`onCreateNode` occurs before `createPages`

Comment (sammartinez) [reply gatsby-node/constants.js]: Got it :)

**Pull request merged**